### ISSUE TITLE
Add server objects to file picker

### ISSUE THREAD

**Issue by minoseah629** [microsoft/azuredatastudio#11654]
**Is your feature request related to a problem? Please describe.**
In VS Code, user can hit control+p to quickly search for the files. 

![image](https://user-images.githubusercontent.com/1980068/89439009-90b19c00-d717-11ea-9629-525d745754c6.png)

Also in VS Code, user can type in left panel and it searchs for file matching search criteria like below: 

![image](https://user-images.githubusercontent.com/1980068/89439314-03bb1280-d718-11ea-918a-81f6effd291d.png)

In ADS, Control + P searches files.  
Typing in left panel does nothing

The closest existing feature in ADS is right click on the server or database and see this view:
databases:
![image](https://user-images.githubusercontent.com/1980068/89442176-5b5b7d00-d71c-11ea-9a80-8e4417aa682d.png)
tables:
![image](https://user-images.githubusercontent.com/1980068/89444395-a4610080-d71f-11ea-903e-0f9d0f1b4aec.png)

**Describe the solution or feature you'd like**
incorporate the control + p to use the search filters shown in screenshots above to allow for search for tables with t: tablename, etc for other database objects.

only allow active connections

**Describe alternatives you've considered**
the forementioned existing features

**Comment by Charles-Gagnon** [comment]
Closing this as a duplicate of https://github.com/microsoft/azuredatastudio/issues/8052

**Comment by minoseah629** [comment]
@Charles-Gagnon, this issue you mentioned does not say anything about putting in the control + p thing is.  i dont know what to call this control.

**Comment by Charles-Gagnon** [comment]
What are you suggesting that should do? Because as you mentioned it does currently search for files open in the workspace/folder. Is there further enhancement to that picker that you'd like to see?

**Comment by minoseah629** [comment]
> What are you suggesting that should do? Because as you mentioned it does currently search for files open in the workspace/folder. Is there further enhancement to that picker that you'd like to see?

I assume the OE Tree View is this left panel: 
![image](https://user-images.githubusercontent.com/1980068/89467276-434b2400-d743-11ea-9ccf-9b46e70dd675.png)

thats good.  

but I asking to have the same features in the control + p : 
![image](https://user-images.githubusercontent.com/1980068/89467532-c10f2f80-d743-11ea-82a9-09f9e3deef6f.png)

the control + p feature specifically currently does files. 

include the database objects and servers (if currently active connection) to this feature.  

s:servername
t:tablename
f:functionname
v:viewname

**Comment by Charles-Gagnon** [comment]
Gotcha, I updated the name for that portion of your ask as a separate issue from the OE filter. As for that one - stay tuned =)

**Comment by minoseah629** [comment]
thanks @Charles-Gagnon .  

the title just has server object.  

just to be sure: does that include tables, functions, views, etc?

**Comment by Charles-Gagnon** [comment]
Yeah - "server objects" here is including all objects on a server. 

Appreciate the feedback!
